### ISSUE TITLE
Panic in `ddev config`

### ISSUE THREAD

**Issue by rfay** [ddev/ddev#5075]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Output of `ddev debug test`

Colima; I don't think anything matters

### Expected Behavior

No panic while testing 
* https://github.com/ddev/ddev/pull/5042

However, the panic exists with just upstream/master right now, a22c46342cb9666a69102c7fc10b63dd4a78ea35

### Actual Behavior

```
mkdir c && cd c
ddev config --project-type=craftcms --docroot=web --create-docroot --upload-dir=cpresources
Flag --upload-dir has been deprecated, please use --upload-dirs instead
You are reconfiguring the project at /Users/rfay/workspace/c.
The existing configuration will be updated and replaced.
Created docroot at /Users/rfay/workspace/c/web
You have specified a project type of 'craftcms' but no project of that type is found in /Users/rfay/workspace/c/web
Could not write settings file: Mutagen sync session 'c' does not exist
Configuration complete. You may now run 'ddev start'.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x1d8 pc=0x104b28078]

goroutine 1 [running]:
github.com/ddev/ddev/pkg/ddevapp.(*DdevApp).GetOmittedContainers(...)
	/Users/rfay/workspace/ddev/pkg/ddevapp/ddevapp.go:372
github.com/ddev/ddev/pkg/ddevapp.(*DdevApp).TrackProject(0x0)
	/Users/rfay/workspace/ddev/pkg/ddevapp/amplitude_project.go:36 +0x58
github.com/ddev/ddev/cmd/ddev/cmd.glob..func48(0x1052bd660, {0x140000381c0?, 0x0?, 0x4?})
	/Users/rfay/workspace/ddev/cmd/ddev/cmd/root.go:98 +0xa4
github.com/spf13/cobra.(*Command).execute(0x1052bd660, {0x14000038180, 0x4, 0x4})
	/Users/rfay/workspace/ddev/vendor/github.com/spf13/cobra/command.go:876 +0x57c
github.com/spf13/cobra.(*Command).ExecuteC(0x1052bb5e0)
	/Users/rfay/workspace/ddev/vendor/github.com/spf13/cobra/command.go:974 +0x354
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/rfay/workspace/ddev/vendor/github.com/spf13/cobra/command.go:902
github.com/ddev/ddev/cmd/ddev/cmd.Execute()
	/Users/rfay/workspace/ddev/cmd/ddev/cmd/root.go:152 +0x28
main.main()
	/Users/rfay/workspace/ddev/cmd/ddev/main.go:28 +0xb8
```

**Comment by rfay** [comment]
This was coming from TrackProject(), so is likely fixed by 
* #5065

**Comment by rfay** [comment]
Solved by #5066
